src/routes/root.ts

```diff
@@ -0,0 +1,33 @@
+import { NextFunction, Request, Response } from 'express';
+import { findIdFor, getId } from '../services/id-service.js';
+import { closeDBConnection } from '../middlewares/datasource.js';
+
+export function root(request: Request, response: Response) {
+	const healthcheck = {
+		message: 'Service Up',
+		timestamp: Date.now(),
+	};
+	response.send(healthcheck);
```

Comment:
As good practice we like to `return` here, like `return response.send(healthcheck)`, whenever we are sending a response. This prevents us from interacting with the response after terminating the connection (which throws server errors).